### PR TITLE
docs: slots_per_nodes for PBS & Slurm

### PR DESCRIPTION
**Pull request by phillip-gaisford**
## Description

Describe usage of slots_per_node for PBS & Slurm.

## Test Plan

Builds & reads locally OK.

## Checklist
- [ ] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

### CODE REVIEW

**Comment by phillip-gaisford** [comment]
Screen Shot 2022-10-21 at 1 24 04 PM
Screen Shot 2022-10-21 at 1 24 18 PM

**Review by jerryharrow — APPROVED** [review]

File: docs/reference/reference-interface/job-config-reference.rst [hunk 1]
```diff
@@ -164,3 +164,5 @@ The following configuration settings are supported:
 
 -  ``slurm``: Slurm cluster details may optionally be specified as for :ref:`experiments
    <slurm-config>`.
+
+-  ``pbs``: PBS cluster details may optionally be specified as for :ref:`experiments <pbs-config>`.
```
Comment (rcorujo) [inline docs/reference/reference-interface/job-config-reference.rst:168]: The "as for" doesn't sound right, although I can't see what actually comes after that once it's replaced.  Is "as for" the same thing as "as per" or "as described in"?  If so, those may be a better choice of wording.

File: docs/reference/reference-training/experiment-config-reference.rst [hunk 2]
```diff
@@ -1300,6 +1300,16 @@ The ``slurm`` section specifies configuration options applicable when the cluste
             - --mem-per-cpu=10
             - --exclusive
 
+``slots_per_node``
+   The minimum number of slots required for a node to be scheduled during a trial. If
```
Comment (rcorujo) [inline docs/reference/reference-training/experiment-config-reference.rst:1304]: Is it really the "minimum number"?  It's not like we're going to allocate at least that many per node.  That's the exact number we're going to allocate.  Isn't this really:

The number of slots to be allocated per node during a trial.

File: docs/reference/reference-training/experiment-config-reference.rst [hunk 3]
```diff
@@ -1320,3 +1330,11 @@ The ``pbs`` section specifies configuration options applicable when the cluster
          pbsbatch_args:
             - -p1000
             - -PMyProjectName
+
+``slots_per_node``
+   The minimum number of slots required for a node to be scheduled during a trial. If
```
Comment (rcorujo) [inline docs/reference/reference-training/experiment-config-reference.rst:1335]: Same comment as above.

**Pull request merged**